src/sentry/api/urls.py

```diff
@@ -590,6 +591,11 @@
         PromptsActivityEndpoint.as_view(),
         name="sentry-api-0-prompts-activity",
     ),
+    url(
+        r"^client-state/(?P<category>[^\/]+)/$",
```

Comment:
If this state is for the client, could it be stored on the client? Your server data looks like it could live in localStorage.

Reply:
We also want to share this state across multiple devices. When the user sign up through their phone, we want them to be able to continue on with what they were doing when they switch to a computer.